seawatch_registration/forms.py

```diff
@@ -3,23 +3,12 @@
 from django.contrib.auth.models import User
 from django.db import ProgrammingError
 
-from .models import Position, Profile, ProfilePosition, DocumentType, Document
-
-
-def positions_as_tuples():
-    try:
-        positions = list(Position.objects.all())
-    except ProgrammingError:
-        return {Position('-----'), Position('-----')}
-    result = set()
-    for position in positions:
-        result.add((position, position))
-    return result
+from .models import Position, Profile, ProfilePosition, DocumentType, Document, Skill
 
 
 class ProfilePositionForm(forms.Form):
```

Comment:
Selected profiles are not initialised. Add __init__ function similar to SkillsForm.

Reply:
Es macht Sinn das zu tun. Gehört aber eigentlich nicht zu den Skills. Werde es umsetzen und auf den Branch pushen ;)

Reply:
fixed with https://github.com/innoq/seawatch_planner/pull/24/commits/6d89a441120b2d1e141a07f46b988a476091a0a0